src/components/FiltersList/presenter.jsx

```diff
@@ -35,7 +35,7 @@ import {
 import Dropdown, { DropdownTrigger, DropdownContent } from 'react-simple-dropdown';
 import types from 'const/modelAttributes';
 import filterTypes from 'const/filterTypes';
-import { isBoolean } from 'lodash';
+import { uniqBy } from 'lodash';
```

Comment:
We already discussed this multiple times. In such way you are importing whole lodash